test/gravity/planet_period_squared_is_proportional_to_cube_of_semimajor_axis_test.py

```diff
@@ -0,0 +1,47 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    convert_to,
+    SI,
+)
+from symplyphysics.laws.gravity import (
+    planet_period_squared_is_proportional_to_cube_of_semimajor_axis as law_of_periods
+)
+
+# Description
+## A planet orbits around a star of mass M = 3e30 kg. The semimajor axis of its orbit is a = 3e9 km.
+## Its period of rotation around the star is about 73 years.
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    M = Quantity(3e30 * units.kilogram)
+    a = Quantity(3e9 * units.kilometer)
+    Args = namedtuple("Args", "M a")
+    return Args(M=M, a=a)
+
+
+def test_law(test_args):
+    result = law_of_periods.calculate_rotation_period(test_args.M, test_args.a)
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.time)
+    result_value = convert_to(result, units.year).evalf(3)
+    assert result_value == approx(73.1, 1e-3)
```

Comment:
Writing `assert_equal(result, 73.1 * units.years)` does not work because it converts everything to seconds and the difference is too big

Reply:
You can add your own tolerance: `assert_equal(result, 73.1 * units.years, tolerance=0.01)`